server/api/services/users.service.js

```diff
@@ -26,6 +26,7 @@ class UsersService {
         let rno = roll.split('-')[1];
 
         user['email'] = `${batch}_${year}${rno}@iiitm.ac.in`;
+        user['password'] = await bcrypt.hash(defaultPassword, saltRounds);
```

Comment:
Since `forEach` is a synchronous function, it might happen that a few promises(of the hash function) are not resolved before they're inserted into the database in the next line. 

We could do one of the following to resolve this issue:-
1. Make one hash outside the loop and assign that variable to `user['password']` inside the loop.
2. Make an array of promises with multiple hashed(equal to the number of users), resolve them using `Promise.all` and assign one hash to each of the users using the index of the array.

Reply:
I guess I would prefer the first method as we are assigning the same hash value to each and every user, so resolving an array of promises might not be needed.

Reply:
If we use the second approach, the hash value for each user would be different since bcrypt will use different salts for all of them.

But it shouldn't be a problem assigning the same hash too, you can go ahead with the first approach